content/W2015/W15-event-teaparty.md

```diff
@@ -0,0 +1,20 @@
+Title: Big CSters Tea Party Social
+Date: 2015-01-25 16:00
+Category: Events
+Tags: social
+Slug: tea-party 
+Author: Melissa Tedesco 
+Summary: We'll be meeting up in the CS lounge for a tea party social!
+
+Big and Little CSters, we would like to invite you to our CSters Tea Party!
+
+There will be tea, pastries, and other snacks! Come out and socialize with younger and older WiCS members.
```

Comment:
Oh, uh, also, something about Bring Your Own Mug.

Not sure about "younger" and "older", lol :)